docs/resources/Audit_Log.md

```diff
@@ -29,198 +31,163 @@ Whenever an admin action is performed on the API, an entry is added to the respe
 | ----------- | ------------------------------------------------------------------------------------------------------- | ----------------------------------------------------- |
-| target_id   | ?string                                                                                                 | id of the affected entity (webhook, user, role, etc.) |
-| changes?    | array of [audit log change](#DOCS_RESOURCES_AUDIT_LOG/audit-log-change-object) objects                  | changes made to the target_id                         |
-| user_id     | ?snowflake                                                                                              | the user who made the changes                         |
-| id          | snowflake                                                                                               | id of the entry                                       |
-| action_type | [audit log event](#DOCS_RESOURCES_AUDIT_LOG/audit-log-entry-object-audit-log-events)                    | type of action that occurred                          |
-| options?    | [optional audit entry info](#DOCS_RESOURCES_AUDIT_LOG/audit-log-entry-object-optional-audit-entry-info) | additional info for certain action types              |
-| reason?     | string                                                                                                  | the reason for the change (0-512 characters)          |
+| target_id   | ?string                                                                                                 | ID of the affected entity (webhook, user, role, etc.) |
+| changes?    | array of [audit log change](#DOCS_RESOURCES_AUDIT_LOG/audit-log-change-object) objects                  | Changes made to the target_id                         |
+| user_id     | ?snowflake                                                                                              | User or app that made the changes                     |
+| id          | snowflake                                                                                               | ID of the entry                                       |
+| action_type | [audit log event](#DOCS_RESOURCES_AUDIT_LOG/audit-log-entry-object-audit-log-events)                    | Type of action that occurred                          |
+| options?    | [optional audit entry info](#DOCS_RESOURCES_AUDIT_LOG/audit-log-entry-object-optional-audit-entry-info) | Additional info for certain action types              |
+| reason?     | string                                                                                                  | Reason for the change (1-512 characters)              |
+
+> warn
+> For `APPLICATION_COMMAND_PERMISSION_UPDATE` events, the `target_id` is the app ID since the `changes` array may contain several permission changes for different individual entities
```

Comment:
```suggestion
> For `APPLICATION_COMMAND_PERMISSION_UPDATE` events, the `target_id` is the app ID since the `changes` array may contain several permission changes for different individual entities.
```